src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -79,9 +79,23 @@ bool OutputStateMachineEngine::ActionExecute(const wchar_t wch)
     case AsciiChars::SO:
         _dispatch->LockingShift(1);
         break;
-    default:
+    case AsciiChars::SUB:
+        // The SUB control is used to cancel a control sequence in the same
+        // way as CAN, but unlike CAN it also displays an error character,
+        // typically a reverse question mark.
+        _dispatch->Print(L'\u2E2E');
+        break;
+    case AsciiChars::DEL:
+        // The DEL control can sometimes be translated into a printable glyph
+        // if a 96-character set is designated, so we need to pass it through
+        // to the Print method. If not translated, it will filtered out there.

```

Comment:
```suggestion
        // to the Print method. If not translated, it will be filtered out
        // there.
```

I wrapped it because under-filling was preferable to breaking the perfect right column symmetry of the comment by going wider... :)